app/models/mixins/ownership_mixin.rb

```diff
@@ -10,10 +10,42 @@ module OwnershipMixin
     virtual_column :evm_owner_userid,                     :type => :string,     :uses => :evm_owner
     virtual_column :owned_by_current_user,                :type => :boolean,    :uses => :evm_owner_userid
     virtual_column :owning_ldap_group,                    :type => :string,     :uses => :miq_group
-    virtual_column :owned_by_current_ldap_group,          :type => :boolean,    :uses => :owning_ldap_group
+
+    # Determine whether to return objects owned by the current user's miq_group
+    # or not.
+    #
+    # Resulting SQL:
+    #
+    #   ((SELECT (LOWER("miq_groups"."description") = 'some_miq_group')
+    #     FROM "miq_groups"
+    #     WHERE "miq_groups"."id" = "THIS_MODELS_TABLE"."miq_group_id"))
+    #
+    # Will result in the following when used with MiqExpression:
+    #
+    #   WHERE (((SELECT (LOWER("miq_groups"."description") = 'some_miq_group')
+    #            FROM "miq_groups"
+    #            WHERE "miq_groups"."id" = "THIS_MODELS_TABLE"."miq_group_id")) = 'true')
+    virtual_attribute :owned_by_current_ldap_group, :boolean, :arel => (lambda do |t|
+      group_tbl  = MiqGroup.arel_table
+      ldap_group = User.current_user.try(:ldap_group).to_s.downcase
+      group_sel  = t.grouping(group_tbl[:description].lower.eq(ldap_group))
+      where_cond = _miq_group_primary_key.eq(_miq_group_foreign_key)
```

Comment:
Yeah, there probably is a less abstract-y way of doing this where I can just hard code some stuff into place and avoid the class methods.  Was keeping it close to the `virtual_delegate` arel that I ripped this off from, but I will try some tweaks now that rebasing this branch isn't a nightmare.